lib/spreadsheet_exporter/xlsx.rb

```diff
@@ -1,43 +1,182 @@
 require 'write_xlsx'
 require_relative 'spreadsheet'
+require "active_support"
+require "active_support/core_ext/hash/keys"
 
 module SpreadsheetExporter
   module XLSX
-    def self.from_objects(objects, options = {})
-      spreadsheet = Spreadsheet.from_objects(objects, options).compact
-      from_spreadsheet(spreadsheet)
+    extend Writexlsx::Utility # gets us `xl_rowcol_to_cell` and `xl_col_to_name`
+
+    ROW_MAX = 65_536 - 1
```

Comment:
Modern Excel can actual handle 1,048,576 rows but that was causing issues with the gem, so I chose the old Excel standard of 65536